boards/arc/axs/doc/index.rst

```diff
@@ -0,0 +1,197 @@
+.. _axs:
+
+DesignWare(R) ARC(R) AXS Software Development platform
+######################################################
+
+Overview
+********
+
+The DesignWare(R) ARC(R) Software Development Platforms are a mainboard
+with a CPU card that provide complete, standalone platforms for software
+development, debugging, and profiling. Synopsys currently has two Software
+Development Platforms available: AXS101 and AXS103
```

Comment:
Do we plan Zephyr support for AXS101? Or would the support here be for AXS103 only?
Maybe use axs103 as name to make clear e.g. the ARC600/700 on axs101 are not planned to be supported.

Reply:
currently, we only support AXS103, I will clarify it.

Reply:
I guess AXS101 makes no sense here because it is based on ARCompact cores (ARC700 & ARC600) which we don't support in Zephyr. So indeed AXS103 might be a better name for the Zephyr board.